R/connectivity-tools.R

```diff
@@ -208,5 +208,5 @@ generate_session_id <- function() {
 }
 
 parse_val <- function(val) {
-  jsonlite::fromJSON(ifelse(is.null(val), "\"\"", val))
+  rjson::fromJSON(ifelse(is.null(val), "\"\"", val))
```

Comment:
Ok, I made this change because it was already using rjson in other parts of the code and had to settle on one library.

Before the library cleanup we were already using more than 20 (which triggers a warning on R CMD check)